src/utils/lbann_library.cpp

```diff
@@ -179,6 +179,10 @@ model *build_model_from_prototext(int argc, char **argv,
                                    pb.model());
     model->setup(io_thread_pool);
 
+    if(opts->has_bool("disable_background_io_activity") && opts->get_bool("disable_background_io_activity")) {
+      model->allow_background_io_activity(false);
```

Comment:
You don't need has_bool(); The code was changed; instead of throwing an exception if the option doesn't exist, 'false' is returned. (other get_XX() methods still throw exceptions, since there's no reasonable default value)

Reply:
Thanks.